packages/airnode-deployer/src/cli/index.ts

```diff
@@ -6,31 +6,33 @@ import isEmpty from 'lodash/isEmpty';
 import uniq from 'lodash/uniq';
 import { hideBin } from 'yargs/helpers';
 import { CloudProvider, version as getNodeVersion } from '@api3/airnode-node';
+import { logger as loggerUtils } from '@api3/airnode-utilities';
```

Comment:
This file now has two loggers? Can we consolidate into a single one?

Reply:
Sure, that was my next step but I figured it'd be best to start with this in this PR. I'm happy to do this now.